packages/r/react-motion.json

```diff
@@ -15,22 +13,21 @@
     "transition",
     "ui"
   ],
-  "author": "chenglou",
+  "author": "",
```

Comment:
should the ci warn about empty author

Reply:
Probably worth doing. This was an automated PR from the cli tool, so I guess they have no author in the NPM package but we've manually set it in the past.